src/nwb_datajoint/common/common_curation.py

```diff
@@ -100,50 +95,51 @@ def make(self, key):
                                              sort_interval=sort_interval)
         AnalysisNwbfile().add(key['nwb_file_name'], key['analysis_file_name'])       
 
-        # add sorting to Sortings
-        Sortings.insert1({'nwb_file_name': key['nwb_file_name'],
-                          'recording_id': key['recording_id'],
-                          'sorting_id': key['sorting_id'],
-                          'sorting_path': key['sorting_path'],
-                          'parent_sorting_id': key['parent_sorting_id']}, skip_duplicates=True)
+        # add sorting to Sorting
+        sorting_key = (SpikeSorting & key).proj().fetch1()
+        sorting_key['sorting_id'] = 'S_'+str(uuid.uuid4())[:8]
+        sorting_key['sorting_path'] = key['sorting_path']
+        sorting_key['parent_sorting_id'] = key['sorting_id']
+
+        SortingID.insert1(sorting_key, skip_duplicates=True)
```

Comment:
Again, I'm not sure if this is recommended datajoint practice, to insert an item into a different table as part of make()

Furthermore, since the sorting_id is random, how is the user going to retrieve this sorting. Or, if they can retrieve it, how will they know this is the automatically curated one?